FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -130,25 +131,44 @@ - (BOOL)compareReferenceImage:(UIImage *)referenceImage
                         error:(NSError **)errorPtr
 {
   BOOL sameImageDimensions = CGSizeEqualToSize(referenceImage.size, image.size);
-  if (sameImageDimensions && [referenceImage fb_compareWithImage:image tolerance:tolerance]) {
+  if (sameImageDimensions && [referenceImage fb_matchesImage:image]) {
     return YES;
   }
   
+  CGFloat percentDifference = CGFLOAT_MAX;
```

Comment:
`CGFloat percentDifference;` 

This confused for a bit and `CGFLOAT_MAX` doesn't add any value